packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/simulationservice/SimulationRequestController.java

```diff
@@ -72,14 +73,11 @@ private static class SimulationRequestBody<T> {
 
 	@GetMapping("/{id}")
 	@Secured(Roles.USER)
+	@HasProjectAccess
```

Comment:
They are no `level` specifiy because by default is READ only?
If so, why not have different annotation? like:
- `@HasProjectAccessRead`
- `@HasProjectAccessWrite`
- `@HasProjectAccessOwner`
- etc.

Reply:
In theory, this is more cleanly used where `@HasProjectAccess` is set on the entire class, then you just override a handful of endpoints with write access.
However we have a number of endpoints on each controller which have no permission checks so it became easier to just annotate each endpoint. In another pass im going to see whether the endpoints that don't have permissions were oversights or on purpose. There are also a number of endpoints which clearly had the wrong permissions before (we were checking for READ on POST/PUTs, etc).  I changed a couple of those but realized I should upgrade those at the same time as the other changes as a 'permission enhancements' PR